hybrasyl/Enums.cs

```diff
@@ -353,11 +353,11 @@ public enum ItemDropType
         [Flags]
         public enum PlayerFlags : int
         {
-            Alive = 0x01,
-            InExchange = 0x20,
-            InDialog = 0x40,
-            Casting = 0x100,
-            Pvp = 0x200,
+            Alive = 1,
+            InExchange = 2,
+            InDialog = 4,
+            Casting = 8,
+            Pvp = 16,

```

Comment:
Why is this enum being reconfigured?

Reply:
For clarity and to make sure it aligns cleanly.

Reply:
Fair enough